package.json

```diff
@@ -11,26 +11,24 @@
     "cli": "node node_modules/react-native/local-cli/cli.js",
     "start": "yarn run ios",
     "test": "jest",
-    "lint": "standard",
-    "commit": "commit-wizard"
+    "lint": "",
```

Comment:
Why remove standard? You can use semistandard if you like semicolons, that's what we generally use in parity-js.

Reply:
have you tried https://github.com/prettier/prettier ? I'm in favor to use it both for formatting and as a linter

Reply:
Yes, I actually use it as a vscode extension, I really like it. Sure, it can work for formatting + linting.